### PR TITLE
The Washington Post | Issue #55

### PR DESCRIPTION
**Pull request by tentpoles**
### Description:

Kindly check my recent PR

### Issue Reference

https://github.com/mazipan/login-page-css/issues/55

### Screenshoot

Done

### Todos

- [x] Add screenshots on `screenshoots` directory
- [x] Update the list in `readme.md`
- [x] Add in `src/index.html`
- [x] Change the author text and link

### Minimum Support

- [x] Click 🌟 button to this repo
- [x] Follow the Author

### Consider to Support

- 👉 🇮🇩 [Trakteer](https://trakteer.id/mazipan?utm_source=github)
- 👉 🌍 [BuyMeACoffe](https://www.buymeacoffee.com/mazipan?utm_source=github)
- 👉 🌍 [Paypal](https://www.paypal.me/mazipan?utm_source=github)
- 👉 🌍 [Ko-Fi](https://ko-fi.com/mazipan)

### CODE REVIEW

**Review by tentpoles — COMMENTED** [review]
Fixed

File: src/16-the-washington-post/index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <footer class="franklin-font">
+        <a href="/login-page-css/"> Back to Home </a>
+        <span class="author">
+          Crafted with ❤️ by <a href="https://github.com/tentpoles" rel="noopener noreferrer" target="_blank">Bhimo Al Kautsar</a> © 2021
+        </span>
+        <a href="https://github.com/mazipan/login-page-css" rel="noopener noreferrer" target="_blank"> Github Repo </a>
+    </footer>
+    <!-- ADDITIONAL CODE -- end YOU MIGHT NOT NEEDED -->
+    <script>
+        const clearInput = () => {
```
Comment (mazipan) [inline src/16-the-washington-post/index.html:89]: use `var` instead, also do not use arrow function here

**Review by mazipan — CHANGES_REQUESTED** [review]

File: src/16-the-washington-post/index.html (hunk 1 above)
Comment (tentpoles) [reply src/16-the-washington-post/index.html:89]: I'll be back soon

**Review by tentpoles — COMMENTED** [review]
Here we go!

**Review by mazipan — APPROVED** [review]

**Comment by mazipan** [comment]
@all-contributors please add @tentpoles for code

**Pull request merged**